cmd/pprof-server/main.go

```diff
@@ -120,3 +144,24 @@ type hostAddr string
 
 func (a hostAddr) Network() string { return "" }
 func (a hostAddr) String() string  { return string(a) }
+
+func kubeClient(inCluster bool, master, kubeconfig string) (*kubernetes.Clientset, error) {
+	var config *rest.Config
+	var err error
+
+	if inCluster {
+		events.Log("kubernetes: using pod service account with InClusterConfig.")
+		config, err = rest.InClusterConfig()
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		events.Log("kubernetes: kubeconfig file.")
+		config, err = clientcmd.BuildConfigFromFlags(master, kubeconfig)
+		if err != nil {
```

Comment:
You could avoid repeating this `if err != nil` if you were moving it after the if/else block.

Reply:
done.